assets/js/common/Table/search.js

```diff
@@ -0,0 +1,23 @@
+const canonicalise = (val) => val.normalize().trim().toLowerCase();
+
+const stringContains = (subject, predicate) =>
```

Comment:
This is not really a predicate

```suggestion
const stringContains = (string, substring) =>
  regularize(string).includes(regularize(substring));
```